core/src/io/anuke/mindustry/type/StatusEffect.java

```diff
@@ -60,8 +60,8 @@ protected void trans(Supplier<StatusEffect> effect, TransitionHandler handler){
         transInit.add(new Object[]{effect, handler});
     }
 
-    @SuppressWarnings("unchecked")
-    protected void opposite(Supplier... effect){
+    @SafeVarargs
```

Comment:
This annotation can't be used in Android 19-14, which Mindustry supports.

Reply:
Bummer.